L73-java-binderchannel/wireformat.md

```diff
@@ -147,6 +148,7 @@ rpc transaction = flags, sequence no, rpc data;
     included.
 *   FLAG_MESSAGE_DATA_IS_PARCELABLE (0x40) - Indicates the message is a parcelable
     object.
+*   FLAG_MESSAGE_DATA_IS_PARTIAL (0x80) - Indicates the message is partial and the receiver should continue reading.
```

Comment:
Proposed change to make this clearer:
Where a message doesn't fit into a single transaction, the message will be sent in multiple transactions with this bit set on all but the final transaction of the message.

Reply:
Done.